### PR TITLE
String operator_plus fix

### PR DESCRIPTION
**Pull request by RameshRavone**

### CODE REVIEW

File: include/core/Godot.hpp [hunk 1]
```diff
@@ -50,15 +50,14 @@ class GodotScript {
...
-
 #define GODOT_CLASS(Name) \
 	public: inline static char *___get_type_name() { return (char *) #Name; } \
 	private:
 
 #define GODOT_SUBCLASS(Name, Base) \
 	public: inline static char *___get_type_name() { return (char *) #Name; } \
 	inline static char *___get_base_type_name() { return (char *) #Base; } \
-	inline Name(godot_object *o) : Base(o) {} \
+	//inline Name(godot_object *o) : Base(o) {} \
```
Comment (karroffel) [inline include/core/Godot.hpp]: Why is that here?

Comment (RameshRavone) [inline include/core/Godot.hpp]: It fixed the class Inheritance
